.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -9,10 +9,15 @@ assignees: ''
 
 ## Pre-release
 
+- [ ] Create a [new project] for the next release version
+- [ ] Add build targets for the new release on [Docker Hub]
+  - [cilium](https://hub.docker.com/repository/docker/cilium/cilium/builds/edit)
+  - [operator](https://hub.docker.com/repository/docker/cilium/operator/builds/edit)
+  - [docker-plugin](https://hub.docker.com/repository/docker/cilium/docker-plugin/builds/edit)
 - [ ] Check that there are no [release blockers] for the targeted release version
 - [ ] Ensure that outstanding [backport PRs] are merged
-- [ ] Create a [new project] for the next release version
-  - [ ] Move any unresolved issues/PRs into the newly created release project
+- [ ] Move any unresolved issues/PRs from old release project into the newly
+      created release project
```

Comment:
Should we also list to have the runtime image rebuilt/updated in here as part of the process?

Reply:
Could be an option, it's clearly gotten out-of-date in the past. I wonder if that's more appropriate for just a periodic bump that for example, gets automatically posted against the repository.

I guess there's no harm in adding this into the list to check off (even if we skip it because there's no change)